src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostClientServiceFactory.SolutionChecksumUpdater.cs

```diff
@@ -146,27 +160,22 @@ private void OnWorkspaceChanged(object sender, WorkspaceChangeEventArgs e)
                 }
             }
 
-            private void CreateInitialSolutionChecksum()
+            private async Task SynchronizeAssets()
             {
-                // initial solution checksum creation won't be affected by global operation.
-                // cancellation can only happen if it is being shutdown.
-                Task.Run(async () =>
-                {
-                    await UpdateSolutionChecksumAsync(ShutdownCancellationToken).ConfigureAwait(false);
+                _synchronize = false;
 
-                    var remoteHostClient = await _service.GetRemoteHostClientAsync(ShutdownCancellationToken).ConfigureAwait(false);
-                    if (remoteHostClient == null)
-                    {
-                        return;
-                    }
+                var remoteHostClient = await _service.GetRemoteHostClientAsync(ShutdownCancellationToken).ConfigureAwait(false);
+                if (remoteHostClient == null)
```

Comment:
I guess in current design it can't but there is already talk going on to make remote host enabled/disabled in the middle of VS session. code lens, test impact all want that.